vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/RoundExpressionProcessor.java

```diff
@@ -21,16 +21,26 @@
 import java.util.Optional;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import javax.inject.Named;
 
+import org.apache.commons.lang3.EnumUtils;
 import org.apache.commons.lang3.math.NumberUtils;
 
 @Named
 public class RoundExpressionProcessor implements IExpressionProcessor
 {
-    private static final Pattern ROUND_EXPRESSION_PATTERN = Pattern.compile(
-            "^round(?:\\((-?\\d+(?:\\.\\d*)?)(?:,\\s*(\\d+))?\\))$", Pattern.CASE_INSENSITIVE);
+    private static final Pattern ROUND_EXPRESSION_PATTERN;
+
+    static
+    {
+        String roundingModes = EnumUtils.getEnumList(RoundingMode.class).stream().map(e -> e.name().toLowerCase())
```

Comment:
`EnumUtils.getEnumList(RoundingMode.class).stream()` -> `Stream.of(RoundingMode.values)`